### DISCUSSION TITLE
2024-06-03(월)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#82] [데일리스크럼]
아래 내용을 복사해서 작성해주시면 됩니다.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일
signup 페이지 코드리뷰 반영
커스텀 인풋 공통 컴포넌트 구현

### 해야할 일
onboarding 페이지 UI 구현

### 어려운 일
이번주 안에 페이지 작업을 끝낼 수 있을까요...(남은 페이지: onboarding / my_order / my_review_write / my_info)

**Reply by devwqc** [reply]
작업해야 하는 페이지가 그렇게 많나요...? 이번 주에 완료하지 못하더라도 괜찮으니까 부담 없이 하세요~

**Comment by minyoung0503** [comment]
### 완료한 일
대표 리뷰 컴포넌트 -> 회의끝나고 PR 올릴 예정

### 해야할 일
전체 리뷰 페이지

### 어려운 일
집중이 왜이렇게 안 될까요 ...

**Reply by devwqc** [reply]
저도 갑자기 집중이 잘 안되는데
저희가 스프린트 막바지까지 오기도 했고
계속 새로운 것들 배우면서 쉼 없이 프로젝트 진행해서 그러지 않을까 싶어요...

쉴 때는 프로젝트 걱정하지 말고 푹 쉬면서 스트레스도 푸시고 이제 조금 남았으니까 마지막까지 파이팅!!💪💪

**Comment by seolsis** [comment]
### 완료한 일
- 결제연동
### 해야할 일
- UI
### 어려운 일
- 저도 집중 안대오.
- 결제연동 사이즈조절

**Reply by devwqc** [reply]
헉... 지원님도 집중 잘 안되시는군요...

이제 백엔드 작업 들어가면 바빠져서 집중하기 싫어도 하게 되지 않을까 싶어요!
지원님이 맡으신 부분들 충분히 어려운 작업이니까 천천히 진행해 보시고 너무 어려우면 언제든 말씀해 주세요👍

**Comment by devwqc** [comment]
### 완료한 일
- 배너 캐러셀 컴포넌트 구현
- 카테고리 버튼 컴포넌트 구현

### 해야할 일
- 스크롤 Top 버튼 컴포넌트 구현
- Nav + 레이아웃 최종 작업
- 홈 페이지 및 상품 목록 리스트

### 어려운 일
- 집중이 많이 떨어짐...
